app/views/users/show.html.erb

```diff
@@ -1,9 +1,22 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <% unless current_user == @user %>
```

Comment:
- [ ] Same as above, you'll be also able to avoid code repetition and reuse the helper method her by refactoring view logic.